docs/api.md

```diff
@@ -96,11 +124,36 @@ Mixins that are bound to several different types can find the current API type i
 or `type` members. [`this.type`](#apitype) would for instance return `GenericObject` and
 [`this.genericType`](#apigenerictype) would return `barchart`.
 
-See the [Mixins examples](/examples/README.md#mixins) on how to use it.
+See the [Mixins examples](/examples/README.md#mixins) on how to use it, below is an outline of what the mixin
```

Comment:
How about adding a small section on the order of which mixins are applied. Or is that obvious?